project/Build.scala

```diff
@@ -57,13 +57,29 @@ object Scrooge extends Build {
       "sonatype-public" at "https://oss.sonatype.org/content/groups/public"
     ),
 
-    ScoverageSbtPlugin.ScoverageKeys.coverageHighlighting := true,
+    ScoverageKeys.coverageHighlighting := true,
 
     libraryDependencies ++= Seq(
-      "org.scalatest" %% "scalatest" % "2.2.4" % "test",
-      "org.scalacheck" %% "scalacheck" % "1.12.2" % "test",
+      scalatestLib(scalaVersion.value) % "test",
+      scalacheckLib(scalaVersion.value) % "test",
       "junit" % "junit" % "4.12" % "test"
     ),
+
+    libraryDependencies := {
```

Comment:
instead of this, can we explicitly request scoverage 2.11 in the addSbtPlugin section?  Here's an example of what might be a simpler way to do this:

https://github.com/sbt/sbt/blob/83f35b18d348d36611570642833976d0f6520a60/sbt/src/sbt-test/dependency-management/override2/build.sbt#L3